### DISCUSSION TITLE
Is there still a need for snowballing when using ASReview?

### DISCUSSION THREAD

**Discussion by LHofstee** [asreview/asreview#643] [Q&A] [answered]
When making use of ASReview, the search could be made more broad. Is there than still a need to use snowballing after the screening phase?

**Comment by boer0107-zz** [comment, marked as answer]
The purpose of snowballing is mainly to find articles that you did not find with your database search. Because ASReview enables you to screen more papers you can broaden your search and miss less papers, But you will always miss something. No search is perfect and above all, not everything is covered by databases, for instance older publications, articles in lesser known journals or grey literature (e.g. reports).

**Reply by boer0107-zz** [reply]
Further reading: van Haastrecht M, Sarhan I, Yigit Ozkan B, Brinkhuis M and Spruit M (2021) SYMBALS: A Systematic Review Methodology Blending Active Learning and Snowballing. Front. Res. Metr. Anal. 6:685591. doi: [10.3389/frma.2021.685591](https://doi.org/10.3389/frma.2021.685591)

**Reply by MaxvanHaastrechtUU** [reply]
Agreed with @boer0107, snowballing certainly serves a purpose even when using a database search.

Backward snowballing can help to uncover grey literature. Besides the examples mentioned already, another set of contributions generally not included in databases are bachelor and master theses. At times the quality of theses can be better than classical published research, so it is a shame to completely disregard such research.

One of the big issues that still remains with backward snowballing is adequate tool support. If anyone has suggestions in this regard they are always welcome. Otherwise it may be a nice idea for a research project. One tool I know of is this: https://github.com/JoaoFelipe/snowballing, but there are clearly opportunities for improvement. The authors of the tool do have a nice paper on 'hybrid search strategies' (i.e. combining database search and snowballing): https://doi.org/10.1016/j.infsof.2020.106294.

Forward snowballing is perhaps less obvious when also performing an extensive database search, but one interesting application there is to use forward snowballing to update the review: https://doi.org/10.1016/j.infsof.2020.106366. Regardless of whether you intend to apply this approach, the paper is an interesting read. The main author is Claes Wohlin, the godfather of snowballing.
